Recon/src/Recon/OverlayProducer.cxx

```diff
@@ -118,6 +118,7 @@ void OverlayProducer::produce(framework::Event &event) {
         event.getCollection<ldmx::SimCalorimeterHit>(collName, simPassName_);
     // but don't copy ecal hits immediately: for them, wait until overlay
     // contribs have been added. then add everything through the hitmap
+    // TAV: the string "Overlay" should be configurable too
```

Comment:
Why? A string needs to be added becasue you might (in fact people often do) run sim and overlay in the same pass, so there needs to be a different collection name for the overlay procedure – combining two simhit collections from the same system – to work. Always using "Overlay" makes it very clear what these simhits are. I think making it configurable is asking for trouble.

Reply:
Every (maybe just most?) collection name is configurable in ldmx-sw, and the same argument you used here could be said about those too, like changing the name of the track collection, etc. I dont expect this to be changed, but right now it's "Overlay" inserted into many places, if you change one of them the rest will not follow, which is not a good practice. And at that point, having the consistency with other producers to be able to change the name if needed would be good, or at least "uniform"